src/test/java/com/snowflake/kafka/connector/ProduceConsumeMessageIT.java

```diff
@@ -0,0 +1,45 @@
+package com.snowflake.kafka.connector;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import com.snowflake.kafka.connector.fake.SnowflakeFakeSinkTask;
+import java.util.List;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ProduceConsumeMessageIT extends ConnectClusterBaseIT {
```

Comment:
Do we need to keep this test?

Reply:
As we decided to remove the FakeConnector I removed the test